docs/notebooks/Chirality_Check.ipynb

```diff
@@ -157,9 +157,9 @@
       "text/plain": [
-       "<Figure size 720x576 with 1 Axes>"
+       "<Figure size 576x432 with 1 Axes>"
```

Comment:
I can't comment outside of this, but "$ 4.43 :nbsphinx-math:`times `10^{-8} Å$;" is not being rendered and in "label1="enantiomer1 with orthoation and reflection"," orthoation should be "rotation and reflection" or "orthogonal".

Reply:
This is fixed in ced0538.
Thanks for reviewing carefully.